BUILD.bazel

```diff
@@ -1325,6 +1338,35 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+cc_library(
+    name = "ray_util_with_spdlog",
+    srcs = glob(
+        [
+            "src/ray/util/*.cc",
+        ],
+        exclude = [
+            "src/ray/util/*_test.cc",
+        ],
+    ),
+    hdrs = glob([
+        "src/ray/util/*.h",
+    ]),
+    copts = COPTS_SPDLOG,
+    strip_include_prefix = "src",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":sha256",
+        "//src/ray/protobuf:event_cc_proto",
+        "@boost//:asio",
+        "@boost//:property_tree",
+        "@com_github_google_glog//:glog",
```

Comment:
We still need this for signal handling and stacktrace print right?

Reply:
> We still need this for signal handling and stacktrace print right?

It's indeed now.